__tests__/endpoints.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../app/app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe.only("GET /api/categoriess", () => {
+  it("Returns an error message when endpoint is not found", () => {
+    return request(app)
+      .get("/api/dasfas")
```

Comment:
don't forget to make sure your test description and endpoint match 😄 I know we were just playing around with this!